CMakeLists.txt

```diff
@@ -113,6 +113,12 @@ if(UNWIND_FOUND)
     list(APPEND EXTERNAL_LIBRARIES ${UNWIND_LIBRARIES})
 endif()
 
+#----------------------------------------------------------
```

Comment:
I'm not sure we should do submodule checkouts in CMake.

Reply:
I wrote a lengthy response in the other PR.

What do you propose as an alternative here? Should we attempt to detect the `.git` path, and if it is not present, print a user-friendly error message that advises the user on how to get the dependency.

I agree on principle that putting tools like git inside CMake is not really nice. On the other hand, errors caused by not having submodules available should annoy users as little as possible.

Reply:
The CMake PR has been updated to remove git from CMake.